### PR TITLE
Changed BroadcastBlockBreakAnimation and BroadcastBlockEntity parameters to vectors

### PR DESCRIPTION
**Pull request by Bond-009**

### CODE REVIEW

File: src/BlockEntities/BedEntity.cpp [hunk 1]
```diff
@@ -43,14 +43,12 @@ void cBedEntity::SendTo(cClientHandle & a_Client)
 void cBedEntity::SetColor(short a_Color)
 {
 	m_Color = a_Color;
-	int posX = m_PosX;
-	int posY = m_PosY;
-	int posZ = m_PosZ;
+	Vector3i Pos = GetPos();
```
Comment (madmaxoft) [inline src/BlockEntities/BedEntity.cpp]: Use `auto`

File: src/World.cpp [hunk 2]
```diff
@@ -2420,18 +2420,18 @@ void cWorld::BroadcastBlockAction(int a_BlockX, int a_BlockY, int a_BlockZ, Byte
...
 
 
 
 
 
-void cWorld::BroadcastBlockEntity(int a_BlockX, int a_BlockY, int a_BlockZ, const cClientHandle * a_Exclude)
+void cWorld::BroadcastBlockEntity(Vector3i a_Blockpos, const cClientHandle * a_Exclude)
 {
-	m_ChunkMap->BroadcastBlockEntity(a_BlockX, a_BlockY, a_BlockZ, a_Exclude);
+	m_ChunkMap->BroadcastBlockEntity(a_Blockpos, a_Exclude);
```
Comment (madmaxoft) [inline src/World.cpp]: Capital `P` in `Pos`

**Comment by madmaxoft** [comment]
Fix the cosmetics and in it goes :)

**Comment by Bond-009** [comment]
@madmaxoft Ready :)

**Pull request merged**